src/model/docker.ts

```diff
@@ -28,6 +28,44 @@ const Docker = {
       testMode === 'all' ? ['playmode', 'editmode', 'COMBINE_RESULTS'] : [testMode]
     ).join(';');
 
+    const dockerPaths = new Map([
+      [
+        'linux',
+        new Map([
+          ['shellCommand', '/bin/bash /dist/entrypoint.sh'],
+          ['sshAgent', '/ssh-agent'],
+          ['githubHome', '/root'],
+          ['githubWorkflow', '/github/workflow'],
+          ['githubWorkspace', '/github/workspace'],
+          ['stepsPathParent', `${actionFolder}/steps`],
+          ['stepsPathContainer', '/steps'],
+          ['entrypointPathParent', `${actionFolder}/`],
+          ['entrypointPathContainer', '/dist'],
+          ['knownHostsParent', ' /home/runner/.ssh/known_hosts'],
+          ['knownHostsContainer', '/root/.ssh/known_hosts'],
```

Comment:
This is pretty abstract and not very cohesive.

Someone recently added windows support to unity-builder. It would be helpful to keep the code almost identical because we are working on merging the code.

Reply:
Ah, that's a good point you mentioned; I can take a look at those changes and make them a little more coherent. I based that directory list on what I saw originally present in the original code, but if there's already plans for a set standard in another code change, I'd be more than happy to leverage that for this! 😃